system-tests/README.md

```diff
@@ -39,7 +39,7 @@ From the `MVD` root folder, execute the following command to build the connector
 Then, to bring up the dataspace, please execute the following command from the `MVD` root folder:
 
 ```bash
-docker-compose -f system-tests/docker-compose.yml up --build
+docker compose -f system-tests/docker-compose.yml up --build
```

Comment:
Just as a side note: and this may be for another issue/pr but the `cd.yaml` pipeline still uses `docker-compose`, which has been [deprecated](https://docs.docker.com/compose/compose-v2/). So in order to keep it and the documentation in sync, we probably want to update `cd.yaml` as well...